server/services/timelineService.ts

```diff
@@ -18,11 +13,13 @@ export default class TimelineService {
   async getTimeline(prisonNumber: string, token: string, username: string): Promise<Timeline> {
     try {
       const timelineResponse = await this.educationAndWorkPlanClient.getTimeline(prisonNumber, token)
-      timelineResponse.events = timelineResponse.events.filter(filterTimelineEvents)
 
       const timeline = toTimeline(timelineResponse)
-      timeline.events = await this.addPrisonNameToPrisons(timeline.events, username)
-      return timeline
+      const timelineWithPrisonNamesPopulated: Timeline = {
```

Comment:
I'm not a fan of mutating properties on objects if we can help it (I'm surprised the linter didn't moan TBH)
(it's especially a problem with sorted arrays, and that has caught us out a couple of times elsewhere)